### PR TITLE
Wl mse storage maintenance

### PR DESCRIPTION
**Pull request by stephamon**
takes over #2096

### CODE REVIEW

**Comment by stephamon** [comment]
Hi @kmsholcomb can you edit this article?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-15a3ee69e3/how-to/)

**Comment by kmsholcomb** [comment]
Yep. I'll get to it this afternoon!

**Comment by kmsholcomb** [comment]
Hi @stephamon. Edits are attached. Let me know if you have any questions. Thanks!

[Microsoft Exchange storage maintenance and best practices_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1129310/Microsoft.Exchange.storage.maintenance.and.best.practices_editsKH.pdf)
[Microsoft Exchange storage maintenance and best practices_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1129311/Microsoft.Exchange.storage.maintenance.and.best.practices_editsKH.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-83daf6cd8d/how-to/)

File: content/exchange/microsoft-exchange-storage-maintenance-and-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   Allow at least 15 minutes for the storage quota to be increased.
+
+**Warning:** Most [mail clients](/how-to/cloud-office-support-terminology/#cloud-office-terminology) have maximum storage limits. For example, Outlook 2016 allows a maximum of 50 GB of mail storage. If your Exchange mailbox size exceeds your mail client limit you will experience mail synchronization issues with your mail client.
+
+### Recommendations for users nearing their storage capacity
+
+Check mailbox usage regularly to determine which users are nearing capacity. Following are some recommendations for users nearing their storage capacity:
+
+- Notify users about their capacity and advise them to archive items to free up mailbox storage. Rackspace Email Archiving ensures that email traffic to and from your domain is archived. For more information, see [Enable email archiving: Cloud Office Control Panel](/how-to/enable-email-archiving-cloud-office-control-panel/).
+
```
Comment (DonNateR) [inline content/exchange/microsoft-exchange-storage-maintenance-and-best-practices.md]: We should remove the line breaks between items in this bulleted list to reduce whitespace.

File: content/exchange/microsoft-exchange-storage-maintenance-and-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- Email in the **Deleted Items** folder counts toward the storage quota of the mailbox. To ensure that a deleted message does *not* count toward your mailbox storage, delete the message from the **Deleted Items** folder.
+
+   If you need to recover a message that you deleted from the **Deleted Items** folder, right-click the folder and select **Recover Deleted Items**.
+
+   **Note:** Messages purged from the **Deleted Items** folder can be recovered for up to 14 days after deletion.
+
+### Mailbox storage best practices
+
+- Enable [**Rackspace Cloud Office Archiving**](/how-to/enable-email-archiving-cloud-office-control-panel/) for all domains.   
+
```
Comment (DonNateR) [inline content/exchange/microsoft-exchange-storage-maintenance-and-best-practices.md]: We should remove the line breaks in this list as well, with exception to lines 74 through 84.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9cb0381fe9/how-to/)

**Pull request merged**